include/crow/middlewares/cookie_parser.h

```diff
@@ -30,10 +32,131 @@ namespace crow
 
     struct CookieParser
     {
+        // Cookie stores key, value and attributes
+        struct Cookie
+        {
+            enum class SameSitePolicy
+            {
+                Strict,
+                Lax,
+                None
+            };
+
+            template<typename U>
+            Cookie(const std::string& key, U&& value):
+              Cookie()
+            {
+                key_ = key;
+                value_ = std::forward<U>(value);
+            }
+
+            // format cookie to HTTP header format
+            std::string format() const
```

Comment:
I would change this to `dump()` since that's what every other class in Crow uses to generate a string.